### PR TITLE
Add Terminal class

### PR DESCRIPTION
**Pull request by lainsce**
As seen on faux-terminals thru the UI.

EDIT: was just in the wrong place

### CODE REVIEW

File: src/widgets/_views.scss [hunk 1]
```diff
@@ -17,4 +17,18 @@ scrolledwindow {
 .view {
     background-color: bg-color(1);
     color: $fg-color;
+
+    &.terminal {
+        background-color: #252e32;
+        color: #94a3a5;
+
+        & text {
+            font-family: monospace;
```
Comment (danrabbit) [inline src/widgets/_views.scss:26]: Looks like this actually needs to apply to `.terminal` , not `text` to take effect

Comment (lainsce) [reply src/widgets/_views.scss]: Fixed.

File: src/widgets/_views.scss [hunk 2]
```diff
@@ -17,4 +17,17 @@ scrolledwindow {
...
     background-color: bg-color(1);
     color: $fg-color;
+
+    &.terminal {
+        background-color: #252e32;
+        color: #94a3a5;
+        font-family: monospace;
+
+        text {
+            selection {
```
Comment (danrabbit) [inline src/widgets/_views.scss:27]: can probably either remove `text` or combine this to be `text selection {}`

Comment (lainsce) [reply src/widgets/_views.scss]: Combined they work. Fixed.

**Pull request merged**